Food Ordering System/index.html

```diff
@@ -0,0 +1,186 @@
+  <link rel="stylesheet" href="style.css">
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@200&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css2?family=Dancing+Script:wght@700&display=swap" rel="stylesheet">
+</head>
+
+<body>
+  <div class="header">
+    <div class="logo-nav">
+      <div class="logo">
+        <img src="img/logo1.png" alt="">
+        <p id="eureka">Eureka Bistro</p>
+      </div>
+
+      <nav class="navigation">
+        <ul>
+          <li>HOME</li>
+          <li>ABOUT</li>
+          <li>SERVICE</li>
+          <li>CONTACTS</li>
+          <a onclick="openForm()" id="cart-logo" href="#"></a>
+        </ul>
+      </nav>
+
+      <div class="form-popup" id="my-form">
+        <form action="#" class="form-container">
+          <button id="cart-button" onclick="closeForm()">close</button>
+          <h2>My Order</h2>
+          <hr>
+          <ul id="cart-list">
+          </ul>
+          <div id="shopping-cart">
+            <p>Total: $0</p>
+          </div>
+        </form>
+      </div>
+    </div>
+
+    <div class="header-text">
+      <h1>Ordering Food</h1>
+      <p>Celerius quam asparagi cocunter</p>
+    </div>
+  </div>
+  <div class="selector">
+    <div>
+      <select>
+        <option value="">CATEGORY</option>
+      </select>
+      <select>
+        <option value="">SORTING</option>
+      </select>
+      <select>
+        <option value="">VEGETARIAN</option>
+      </select>
+    </div>
+
+    <div>
+      <label id="price" for="slider"> Price: </label>
+      $9
+      <input type="range" class="slider" name="price" min="15" max="100">
+      $50
+    </div>
+  </div>
+
+  <main>
+    <div class="image">
+      <img src="img/Protein Salad.jpg" alt="">
+      <div class="imageText">
+        <p class="food-name">Protein Salad</p>
+        <p class="food-type">Salads</p>
+        <p class="price">$9.00</p>
+        <a onclick="addButton(0)" href="#" class="add-button" id="1">ADD TO CART</a>
+      </div>
+    </div>
+
+    <div class="image">
+      <img src="img/Vitamin Salad.jpg" alt="">
+      <p class="food-name">Vitamin Salad</p>
+      <p class="food-type">Salads</p>
+      <p class="price">$9.00</p>
+      <a onclick="addButton(1)" href="#" class="add-button">ADD TO CART</a>
+    </div>
+
+    <div class="image">
+      <img src="img/Asparagus Salad.jpg" alt="">
+      <p class="food-name">Asparagus Salad</p>
+      <p class="food-type">Salads</p>
+      <p class="price">$9.00</p>
+      <a onclick="addButton(2)" href="#" class="add-button">ADD TO CART</a>
+    </div>
+
+    <div class="image">
+      <img src="img/Tuna Salad.jpg" alt="">
+      <p class="food-name">Tuna Salad</p>
+      <p class="food-type">Salads</p>
+      <p class="price">$12.00</p>
+      <a onclick="addButton(3)" href="#" class="add-button">ADD TO CART</a>
+    </div>
+
+    <div class="image">
+      <img src="img/Prosciutto and Rucola Salad.jpg " alt="">
```

Comment:
add alternate text to all images

Reply:
A different class name for all of them or an alternate text for each of them ?

Reply:
alt text for images: 

> Alt text (alternative text), also known as "alt attributes", “alt descriptions", or technically incorrectly as "alt tags,” are used within an HTML code to describe the appearance and function of an image on a page.